### PR TITLE
Document SSO Logout Limitation for FusionAuth admin App

### PR DESCRIPTION
**Pull request by spwitt**
Document the FusionAuth admin app logout behavior limitation.

- https://github.com/FusionAuth/fusionauth-site/issues/1346

### CODE REVIEW

File: site/docs/v1/tech/core-concepts/applications.adoc [hunk 1]
```diff
@@ -141,7 +141,7 @@ See the link:/docs/v1/tech/oauth/endpoints#logout[Logout endpoint] for more info
 [field]#Logout behavior# [optional]#Optional#::
 This selector allows you to modify the behavior when using the link:/docs/v1/tech/oauth/endpoints#logout[Logout endpoint] with this Client Id.
 +
-    * All applications - This is the default behavior. Logout out of the FusionAuth SSO, call each registered Logout URLs for the entire tenant and then redirect to the Logout URL registered for this application.
+    * All applications - This is the default behavior. Upon Logout of the FusionAuth SSO, call each registered Logout URLs for the entire tenant and then redirect to the Logout URL registered for this application.
```
Comment (spwitt) [inline site/docs/v1/tech/core-concepts/applications.adoc:144]: I noticed the redundant "Logout out" when I was looking at logout behavior stuff.

File: site/docs/v1/tech/samlv2/index.adoc [hunk 2]
```diff
@@ -198,4 +198,10 @@ Of course, you can modify, delete, or add any attributes you want by configuring
 
 == Limitations
 
+=== IdP Limitations
```
Comment (spwitt) [inline site/docs/v1/tech/samlv2/index.adoc:201]: The Limitations page had these two sections, but only one was displayed on the SAML page. I assumed it was an oversight, but let m know if I should roll this back.

File: site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc [hunk 3]
```diff
@@ -0,0 +1 @@
+An Application's OAuth configuration allows you to specify the logout behavior of an application as either "All applications" or "Redirect only." Specifying the default value of "All applications" will call the registered logout URLs for each application on the Tenant _except_ for the FusionAuth admin application. See the https://github.com/FusionAuth/fusionauth-issues/issues/1699[GitHub issue] for more information.
```
Comment (mooreds) [inline site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc:1]: Please use dashes in include files, except for the initial underscore:

`site/docs/v1/tech/oauth/_logout-behavior-all-applications.adoc`

File: site/docs/v1/tech/reference/limitations.adoc [hunk 4]
```diff
@@ -73,6 +73,10 @@ There are a number of items in FusionAuth created by default and which cannot be
 
 include::docs/v1/tech/shared/_default-entities.adoc[]
 
+== OAuth Logout Behavior
```
Comment (mooreds) [inline site/docs/v1/tech/reference/limitations.adoc:76]: Please add to the TOC at the top of this file.

File: site/docs/v1/tech/samlv2/index.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/samlv2/index.adoc:201]: Thanks, appreciate it.

File: site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc:1]: Also, I've started using a line per sentence, which lets diffs be better.

Comment (mooreds) [reply site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc:1]:
```suggestion
An Application's OAuth configuration allows you to specify the logout behavior of an application as either "All applications" or "Redirect only."
Specifying the default value of "All applications" will call the registered logout URLs for each application on the Tenant _except_ for the FusionAuth admin application.
See the https://github.com/FusionAuth/fusionauth-issues/issues/1699[GitHub issue] for more information.
```

**Review by mooreds — CHANGES_REQUESTED** [review]
A few small changes.

File: site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc (hunk 3 above)
Comment (robotdan) [inline site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc:1]: 1. `"Redirect only."` -> `"Redirect only".`
2. `on the Tenant` -> `in the Tenant`

**Comment by mooreds** [comment]
Also @spwitt , can you please add the limitations section to the SSO guide? https://fusionauth.io/docs/v1/tech/guides/single-sign-on

File: site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc (hunk 3 above)
Comment (spwitt) [reply site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc:1]: [MLA docs](https://style.mla.org/the-placement-of-a-comma-or-period-after-a-quotation/).
> When a comma or period is needed after a quotation, publishers in the United States typically put the punctuation mark before the closing quotation mark.

British publishers put the period outside the quotation marks.

Do you still want me to change that?

File: site/docs/v1/tech/guides/single-sign-on.adoc [hunk 5]
```diff
@@ -698,6 +698,10 @@ Navigate to [breadcrumb]#Applications -> Your Application -> OAuth# and configur
 
 image::guides/single-sign-on/application-config-logout-behavior-docs.png[Configuring the logout behavior for an application.,width=1200,role=top-cropped]
 
+==== Limitations
```
Comment (mooreds) [inline site/docs/v1/tech/guides/single-sign-on.adoc:701]: Can you please make this top level ( `== Limitations`) and add it to the TOC?

**Review by mooreds — APPROVED** [review]
One change only, please add the limitations to the SSO TOC and make it a h1 in the guide. (More comments in that file changes section.)

File: site/docs/v1/tech/guides/single-sign-on.adoc (hunk 5 above)
Comment (spwitt) [reply site/docs/v1/tech/guides/single-sign-on.adoc:701]: I was going for a nesting under Additional Configuration > Logout Behavior > Limitations. I think it's okay to put Limitations at the end as a top level header if you'd prefer.

The page does not have ToC. Should I add one?

Comment (mooreds) [reply site/docs/v1/tech/guides/single-sign-on.adoc:701]: I prefer limitations to be top level (lets everyone know what they are getting into).

That would be awesome if you would add a TOC.

Here's an egrep command which creates one, I typically only have the h1s and h2s in the toc:

```
egrep '^[=]+ ' site/docs/v1/tech/gides/single-sign-on.adoc |sed 's/=//' |sed 's/=/*/g'|sed 's/* /* <</'|sed 's/$/>>/'
```

**Pull request merged**